### ISSUE TITLE
Party disabled, but menu item still active

### ISSUE THREAD

**Issue by SlimeDog** [PaulAnthonyReitz/ReitzMMO#3]
Spigot 1.14.1
ReitzMMO 1.14

```
# This config is used to set all party configurations
Parties_Enabled: false
PartyEXPMaxDistance: 100
```
but GUI still contains item to "Get the party commands!"

**Comment by RitzDaCat** [comment]
https://github.com/PaulAnthonyReitz/ReitzMMO/commit/945ac5dce1bd52c38e2dff9a6e7ec92e7bcf7e53

**Comment by SlimeDog** [comment]
Confirmed fixed in v 1.15. Thanks.
